### DISCUSSION TITLE
is this production ready?

### DISCUSSION THREAD

**Discussion by hiqsociety** [minio/minio#12950] [Q&A]
is this production ready?

1. who is using minio?
2. how can i switch the backend to rocksdb or my custom backend storage layer?
3. how do i use it without container / kubernetes?

**Comment by kannappanr** [comment]
Please take a look at https://docs.min.io/ This will answer most of your questions.

**Reply by hiqsociety** [reply]
ok thx

**Reply by klauspost** [reply]
And the main page contains a selection of users: https://min.io/
